### PR TITLE
Fix: regular expression to parse ping output message

### PR DESCRIPTION
**Pull request by salascha**
While ping command was capture, the parser function was using a wrong regular expresation. the fix add the duplicate token

### CODE REVIEW

File: lib/topology_lib_ping/library.py [hunk 1]
```diff
@@ -30,7 +30,8 @@
 PING_RE = (
     r'^(?P<transmitted>\d+) packets transmitted, '
     r'(?P<received>\d+) received,'
-    r'( \+(?P<errors>\d+) errors,)? '
+    r'( \+(?P<errors>\d+) errors,)?'
+    r'( \+(?P<duplicates>\d+) duplicates,)? '
```
Comment (viswanav) [inline lib/topology_lib_ping/library.py]: Please check for all possible cases of ping response -
Case 1 (Duplicates and Errors) - 10000 packets transmitted, 10000 received, +19 duplicates, +10 errors, 0% packet loss, time 10035ms
Case 2 (Duplicates only) - 10000 packets transmitted, 10000 received, +19 duplicates, 0% packet loss, time 10035ms
Case 3 (Errors only) - 10000 packets transmitted, 10000 received, +10 errors, 0% packet loss, time 10035ms
Case 4 (No duplicates and no errors) - 10000 packets transmitted, 10000 received, 0% packet loss, time 10035ms

**Comment by salascha** [comment]
Please check the latest commit.
the regular expression was validated by using:
(Pdb) mensage_1
'2 packets transmitted, 1 received, +4 duplicates, +6 errors, 50% packet loss, time 999ms'
(Pdb) m = match(PING_RE_8, mensage_1)
(Pdb) m.groupdict().items()
dict_items([('errors', '6'), ('transmitted', '2'), ('duplicates', '4'), ('received', '1'), ('loss_pc', '50'), ('time_ms', '999')])
(Pdb) mensage_2
'10000 packets transmitted, 10000 received, 0% packet loss, time 9999ms'
(Pdb) m = match(PING_RE_8, mensage_2)
(Pdb) m.groupdict().items()
dict_items([('errors', None), ('transmitted', '10000'), ('duplicates', None), ('received', '10000'), ('loss_pc', '0'), ('time_ms', '9999')])
(Pdb) mensage_3
'2 packets transmitted, 1 received, +6 errors, 50% packet loss, time 999ms'
(Pdb) m = match(PING_RE_8, mensage_3)
(Pdb) m.groupdict().items()
dict_items([('errors', '6'), ('transmitted', '2'), ('duplicates', None), ('received', '1'), ('loss_pc', '50'), ('time_ms', '999')])
(Pdb) PING_RE_8
'^(?P<transmitted>\d+) packets transmitted, (?P<received>\d+) received,( \+(?P<duplicates>\d+) duplicates,)?( \+(?P<errors>\d+) errors,)? (?P<loss_pc>\d+)% packet loss, time (?P<time_ms>\d+)ms$'
(Pdb) exit

**Comment by viswanav** [comment]
@salascha Looks good

**Comment by saenzpa** [comment]
Erick, please squash your commits into a single one to avoid extra history added to the repo.

**Comment by salascha** [comment]
Pablo, squash was done successfully.

**Pull request merged**